data/json/monsters/reptile_amphibian.json

```diff
@@ -133,9 +135,10 @@
         "damage_max_instance": [ { "damage_type": "stab", "amount": 8, "armor_multiplier": 0.8 } ]
       }
     ],
-    "reproduction": { "baby_monster": "mon_rattlesnake_s", "baby_count": 13, "baby_timer": 240 },
+    "reproduction": { "baby_monster": "mon_rattlesnake_s", "baby_count": 13, "baby_timer": 180 },
     "baby_flags": [ "AUTUMN", "SUMMER" ],
     "upgrades": { "half_life": 65, "into": "mon_rattlesnake_giant" },
+    "biosignature": { "biosig_item": "shed_snakeskin_big", "biosig_timer": 240 },
```

Comment:
Is this intended? You don't mention changing reproduction timers, and the biosig timer here is off from the others

Reply:
Good catch! That was not intended, the biosig timer is what should have been changed to 180